en/flight_modes/orbit.md

```diff
@@ -0,0 +1,46 @@
+# Orbit (MC)
+
+[<img src="../../assets/site/difficulty_easy.png" title="Easy to fly" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
+
+The *Orbit* flight mode allows you to fly a multicopter (or VTOL in multicopter mode) in a circle, yawing so that it always faces towards the centre.
+
+![Orbit Mode - MC](../../assets/flying/orbit.jpg)
+
+*QGroundControl* (or other compatible GCS or MAVLink API) is *required* to enable the mode, and to set the centre position, initial radius and altitude of the orbit.
+Once enabled the vehicle will fly as fast as possible to the closest point on the commanded circle trajectory and do a slow (1m/s) clockwise orbit on the planned circle, facing the center.
+Instructions on how to start an orbit can be found here: [FlyView > Orbit Location](https://docs.qgroundcontrol.com/en/FlyView/FlyView.html#orbit) (*QGroundControl* guide).
+
+> **Note** RC control cannot be used to start the mode (if you switch to the mode via RC it will sit idle).
+
+Once orbit mode is active, the RC controller can be used to change the orbit altitude, radius, speed, and orbit direction:
+- **Left stick:** up/down for altitude change as in [Position mode](../flight_modes/position_mc.md).
+- **Right stick:** right/left controls speed and direction (counter-clockwise/clockwise), up/down controls orbit radius (smaller/bigger).
+
+The diagram below shows the mode behaviour visually (for a [mode 2 transmitter](../getting_started/rc_transmitter_receiver.md#transmitters-for-aircraft)).
+
+![Orbit Mode - MC](../../images/flight_modes/orbit_MC.png)
+
+The mode can be stopped by switching to any other flight mode (using RC or QGC).
+
+<!--
```

Comment:
Should I describe how it's implemented or an example how to start it via command here?

Reply:
No. For the manual flight modes we tend to explain in bullet form the detail of the way that the sticks control movement here. However this is a "guided" mode, and I don't think we need such a section.

In MAVLink docs we probably should have info on the Orbit Mode protocol as a microservice, but in no way urgent.